packages/xchain-binance/src/client.ts

```diff
@@ -105,6 +116,25 @@ class Client extends BaseXChainClient implements BinanceClient, XChainClient {
     return this.bncClient
   }
 
+  /**
+   * Gets the current network, and enforces type limited to
+   * 'mainnet' and 'testnet', which conflicts with `xchain-client`
+   *
+   * Remove this once @binance-chain has stagenet support.
+   * @returns {Network}
+   */
+  getNetwork(): Network {
+    switch (super.getNetwork()) {
+      case Network.Mainnet:
+        return Network.Mainnet
+      case Network.Testnet:
+        return Network.Testnet
+      default:
```

Comment:
[nitpick] Handle `stagenet`, but not `default` here and at other `switch/case` statement in this file. `default` might be unsafe in case of other updates in `Network` type.